go.sum

```diff
@@ -4,6 +4,8 @@ github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7 h1:YoJbenK9C6
 github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7/go.mod h1:z4/9nQmJSSwwds7ejkxaJwO37dru3geImFUdJlaLzQo=
 github.com/cronokirby/saferith v0.32.0 h1:M7rn0uhYrnZOheLmAZZ5PkXENWVbcQ30mDiPPJbJN58=
 github.com/cronokirby/saferith v0.32.0/go.mod h1:QKJhjoqUtBsXCAVEjw38mFqoi7DebT7kthcD7UzbnoA=
+github.com/cronokirby/saferith v0.33.0 h1:TgoQlfsD4LIwx71+ChfRcIpjkw+RPOapDEVxa+LhwLo=
+github.com/cronokirby/saferith v0.33.0/go.mod h1:QKJhjoqUtBsXCAVEjw38mFqoi7DebT7kthcD7UzbnoA=
```

Comment:
Not very important, but maybe run `go mod tidy` to remove the old version from go.sum?

Reply:
Done